relay-server/src/services/processor.rs

```diff
@@ -2160,10 +2159,10 @@ impl EnvelopeProcessorService {
                 );
 
                 let reason_code = limits.longest().and_then(|limit| limit.reason_code.clone());
-                utils::reject_metrics(
```

Comment:
We still need to call into `reject_metrics`, we still need these outcomes.

Maybe we can call `report_rejected_metrics` and make it not-public, to make sure we always call into `reject_metrics`.

Reply:
Yeah that could be a good way.